README.md

```diff
@@ -13,6 +13,7 @@ access the video decoding features available on your GPU.
 * H.265 (HEVC) - 8 bit, and 10 bit
 * H.264 (AVC) - 8 bit
 * AV1 - 8 bit, and 10 bit
+* VP9 - 8 bit, and 10 bit
```

Comment:
This information also needs to be added to docs/rocDecode-prerequisites.rst

Reply:
Not quite sure why the supported codec type info needs to be in docs/install/rocDecode-prerequisites.rst? @spolifroni-amd

Reply:
Actually should be rocDecode-formats-and-architectures.rst. Updated.

Reply:
The page is named rocDecode-prerequisites.rst

Changing the name entails changing redirects etc. And a Jira issue needs to be opened to track it. If you want the name changed, please open a SWDEV Doc issue for it and send me the issue number. 

In the meantime, please change the name back. Thank you.

Reply:
I'm not sure what this really means. The rocDecode-prerequisites.rst file is in docs/install folder, which does NOT have the supported codec type info. However, in docs/reference/rocDecode-formats-and-architectures.rst, the supported codec types are listed and I added VP9 in this file. The file name is NOT changed. Please review again. @spolifroni-amd